.github/workflows/new-pull-request.yaml

```diff
@@ -21,6 +21,6 @@ jobs:
           issue-number: ${{ github.event.number }}
           body: |
             Thank you for opening a pull request.
-            * A coverage report will be available on [thalia-coverage.s3.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html](https://thalia-coverage.s3.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html) after the "Deploy Coverage Report" job finishes.
-            * Branch specific documentation will be available on [thalia-documentation.s3.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html](https://thalia-documentation.s3.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html) after the "Deploy Documentation" job finishes.
+            * A coverage report will be available on [thalia-coverage.s3.eu-west-1.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html](https://thalia-coverage.s3.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html) after the "Deploy Coverage Report" job finishes.
+            * Branch specific documentation will be available on [thalia-documentation.s3.eu-west-1.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html](https://thalia-documentation.s3.amazonaws.com/${{ env.GITHUB_REF_SLUG }}/index.html) after the "Deploy Documentation" job finishes.
```

Comment:
You changed the link that was shown, but not what is is actually pointing at... @se-bastiaan

Reply:
Lol I'm dumb